<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>519</width>
    <height>419</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox_3">
      <property name="title">
       <string>GroupBox</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Лучшее время:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lb_record">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1" rowspan="2">
     <widget class="QGroupBox" name="groupBox_4">
      <property name="title">
       <string>GroupBox</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Результаты: </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextBrowser" name="circle_Browser"/>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>GroupBox</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Время:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lb_time">
         <property name="text">
          <string>0.0с</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>GroupBox</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="bt_start_stop">
         <property name="text">
          <string>Старт</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="bt_circle">
         <property name="text">
          <string>Круг</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="bt_clear">
         <property name="text">
          <string>Очистка</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>519</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Секундомер</string>
    </property>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>bt_clear</sender>
   <signal>clicked()</signal>
   <receiver>circle_Browser</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>276</x>
     <y>330</y>
    </hint>
    <hint type="destinationlabel">
     <x>312</x>
     <y>247</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bt_clear</sender>
   <signal>clicked()</signal>
   <receiver>lb_record</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>281</x>
     <y>318</y>
    </hint>
    <hint type="destinationlabel">
     <x>98</x>
     <y>129</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
